### PR TITLE
[new modeling strategy] maternal supplementation effect on birthweight exposure

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]

File: docs/source/intervention_models/targeted_bep_and_mms/index.rst [hunk 1]
```diff
@@ -171,71 +171,115 @@ For the implementation of the intervention in an alterative scenarios, we will m
...
+    - Relative to no supplementation 
+    - [Pena-Rosas-et-al-2015]_
+  * - MMN supplemented mothers (overall)
+    - +45.16 (32.31 to 58.02) 
+    - Relative to IFA
+    - Meta-analysis of 13 trials from [Keats-et-al-2019-maternal-supplementation]_, linked as a memo in :ref:`BEP concept model document <2017_concept_model_vivarium_gates_bep>`
+  * - BEP supplemented mothers (undernourished)
+    - +66.96g (13.13 to 120.78)
+    - Relative to MMN
+    - [Ota-et-al-2015]_
```
Comment (yongqx) [inline docs/source/intervention_models/targeted_bep_and_mms/index.rst:220]: @alibow Is that means the birthweight shift = +57.73g if someone takes IFA; and +(57.73 + 45.16)g for MMN, and +(57.73 + 45.16 + 66.96)g for BEP; assuming all three groups compare to no supplementations.

Comment (alibow) [reply docs/source/intervention_models/targeted_bep_and_mms/index.rst:220]: Yep, exactly!

Comment (alibow) [reply docs/source/intervention_models/targeted_bep_and_mms/index.rst:220]: @yongqx, can you please approve this PR if you have finished reviewing? I can merge all of the supplementation PRs after this one is approved. Thanks!

**Review by yongqx — APPROVED** [review]

File: docs/source/intervention_models/targeted_bep_and_mms/index.rst (hunk 1 above)
Comment (alibow) [reply docs/source/intervention_models/targeted_bep_and_mms/index.rst:220]: Thank you!

**Pull request merged**